lib/bigquery/job.js

```diff
@@ -117,12 +118,13 @@ Job.prototype.getMetadata = function(callback) {
  * // Consume the results from the query as a readable stream.
  * //-
  * var through2 = require('through2');
+ * var fs = require('fs');
  *
  * job.getQueryResults(options)
  *   .pipe(through2.obj(function (row, enc, next) {
  *     this.push(JSON.stringify(row) + '\n');
  *   }))
- *   .pipe(fs.createWriteStream('./backup.json'));
+ *   .pipe(fs.createWriteStream('./test/testdata/testfile.json'));
```

Comment:
These actually write to a file (mind you they write nothing)... so having the file exist in our repo helps mitigate it creating additional files. Is this ok?